paper-checkbox.html

```diff
@@ -171,6 +172,7 @@
         vertical-align: middle;
         padding-left: var(--paper-checkbox-label-spacing, 8px);
         white-space: normal;
+        line-height: normal;
```

Comment:
You're setting this on the checkboxLabel though, not the checkbox itself, but it looks right!